samples/net/mqtt/src/modules/transport/credentials_provision/credentials_provision.c

```diff
@@ -0,0 +1,92 @@
+/*
+ * Copyright (c) 2023 Nordic Semiconductor ASA
+ *
+ * SPDX-License-Identifier: LicenseRef-Nordic-5-Clause
+ */
+
+#include <modem/modem_key_mgmt.h>
+
+static const unsigned char ca_certificate[] = {
+#include "ca-cert.pem"
+};
+
+static const unsigned char device_certificate[] = {
+#include "client-cert.pem"
+};
+
+static const unsigned char private_key[] = {
+#include "private-key.pem"
+};
+
+int credentials_provision(void)
+{
+	int err;
+
+	if (sizeof(ca_certificate) > 0) {
```

Comment:
We just accidentally overwrote the CA, because we forgot about that file. Maybe we can warn users if this is overwriting an existing entry?

Reply:
Ill add this as a feature improvement. I don't want to add more functionality at this point. Out of curiosity, did you have an entry accidentally at sec tag 955?